### ISSUE TITLE
Setting convert images to "no" doesn't do anything

### ISSUE THREAD

**Issue by CyberSecutor** [yireo/Yireo_Webp2#132]
When setting the "convert images" option in NextGenImages to No, it keeps converting the images to webp.

![image](https://user-images.githubusercontent.com/843685/188901476-b35d402b-7af4-456a-882e-9468c20f190c.png)

When looking at the code, the config setting `allowImageCreation()` is never used.

```
    public function allowImageCreation(): bool
    {
        return (bool)$this->getValue('yireo_nextgenimages/settings/convert_images');
    }
```

**Comment by jissereitsma** [comment]
Good point. This was indeed added but never implemented. I've implemented the method in the WebP module version 0.12.4, even though I think it requires further finetuning.

I'll close this issue because the feature is now there. But if you bump into issues with it, do feel free to reopen this ticket again.

**Comment by CyberSecutor** [comment]
Thank you Jisse,

I've tested it and it works as expected.
